docs/upgrade.html

```diff
@@ -69,6 +69,13 @@ <h4><a id="upgrade_3_1_0" href="#upgrade_3_1_0">Upgrading to 3.1.0 from any vers
     </li>
 </ol>
 
+<h5><a id="upgrade_300_notable" href="#upgrade_301_notable">Notable changes in 3.1.1</a></h5>
```

Comment:
My html-fu is not particularly strong, but it seems we should change the `id` and `href` tags for the new sections?